rebalancer.py

```diff
@@ -169,18 +173,40 @@ def auto_enable():
         apdays = int(LocalSettings.objects.filter(key='AR-APDays')[0].value)
     else:
         LocalSettings(key='AR-APDays', value='7').save()
-        apdays = 7        
+        apdays = 7
     if enabled == 1:
-        channels = Channels.objects.filter(is_active=True, is_open=True, private=False).annotate(outbound_percent=((Sum('local_balance')+Sum('pending_outbound'))*1000)/Sum('capacity')).annotate(inbound_percent=((Sum('remote_balance')+Sum('pending_inbound'))*1000)/Sum('capacity'))
+        channels = Channels.objects.filter(is_active=True, is_open=True, private=False)
```

Comment:
This will aggregate everything on the remote_pubkey level.

```
channels = Channels.objects.filter(is_active=True, is_open=True, private=False).values('remote_pubkey').annotate(outbound_percent=((Sum('local_balance')+Sum('pending_outbound'))*1000)/Sum('capacity')).annotate(inbound_percent=((Sum('remote_balance')+Sum('pending_inbound'))*1000)/Sum('capacity')).order_by()
```

Reply:
I will make the change and push again. 
Meanwhile please confirm?
we will still require the nested loop for oapD and iapD to get aggregate in and out routing. If we do not aggregate these, one of the channel could be enabled while other is disabled.

Reply:
PS : Why multiplication by 1000 and not 100 to get percent?

Reply:
This was because the database outputs here are integers but wanted to have a resolution of 0.1% increments with proper rounding. There are 2 following lines where another division by 10 happens with a round before truncating the decimal places with `int()`.

Reply:
https://github.com/cryptosharks131/lndg/pull/94/commits/9cc8480cfe8992af4f6abe753f099ba8142a5d95